### PR TITLE
Pages refresh entries

### PR DESCRIPTION
**Pull request by heafalan**
Trying to insert "refreshEntries()" in the method "createEntry()".

### CODE REVIEW

File: packages/pages/src/App.js [hunk 1]
```diff
@@ -68,6 +68,7 @@ export default provideState({
       });
       this.state.name = "";
       this.state.content = "";
+      this.refreshEntries();
```
Comment (julien-f) [inline packages/pages/src/App.js]: `await this.effects.refreshEntries()`

**Comment by heafalan** [comment]
when "refreshEntries()" is called at the end of the method "createEntry", "this.state.entries" is undefined. I don't know why ?

**Comment by julien-f** [comment]
This is a bug, please use `reactlette` instead of `@julien-f/freactal` and it should work :slightly_smiling_face:

File: package.json [hunk 2]
```diff
@@ -18,5 +18,8 @@
     "hooks": {
       "pre-commit": "scripts/lint-staged"
     }
+  },
+  "dependencies": {
+    "reaclette": "^0.6.0"
```
Comment (julien-f) [inline package.json]: Why?

File: packages/cli/package.json [hunk 3]
```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "esm": "^3.0.84",
     "http-request-plus": "^0.6.0",
-    "json-rpc-protocol": "^0.12.0"
+    "json-rpc-protocol": "^0.12.0",
+    "reaclette": "^0.6.0"
   }
```
Comment (julien-f) [inline packages/cli/package.json]: Why?

File: packages/client/package.json [hunk 4]
```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "esm": "^3.0.84",
     "http-request-plus": "^0.6.0",
-    "json-rpc-protocol": "^0.12.0"
+    "json-rpc-protocol": "^0.12.0",
+    "reaclette": "^0.6.0"
```
Comment (julien-f) [inline packages/client/package.json]: Why?

File: packages/server/package.json [hunk 5]
```diff
@@ -18,6 +18,7 @@
     "get-stream": "^4.0.0",
     "json-rpc-protocol": "^0.12.0",
     "koa": "^2.5.3",
-    "koa-static": "^5.0.0"
+    "koa-static": "^5.0.0",
+    "reaclette": "^0.6.0"
```
Comment (julien-f) [inline packages/server/package.json]: Why?

File: packages/pages/package.json [hunk 6]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@julien-f/freactal": "^0.4.0",
```
Comment (julien-f) [inline packages/pages/package.json]: This should no longer be used, please remove it.

**Review by julien-f — APPROVED** [review]

**Pull request merged**